projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/case-edit-submit/case-edit-submit.component.spec.ts

```diff
@@ -641,6 +597,69 @@ describe('CaseEditSubmitComponent', () => {
       const cancelLink = de.query(By.css('a[class=disabled]'));
       expect(cancelLink).toBeNull();
     });
+
+    it('should show event notes when set in event trigger and showEventNotes is called', () => {
+      comp.profile.user.idam.roles = ['caseworker-divorce'];
+      comp.caseFlagField = null;
+      console.log('PROFILE', comp.profile);
```

Comment:
@johnbenjamin-hmcts - can remove this `console.log` statement

Reply:
Thank you for finding this one @Daniel-Lam. I have updated the code now.